modules/ROOT/pages/sap/sap-connector-terminology.adoc

```diff
@@ -0,0 +1,59 @@
+= SAP Terminology
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following are commonly used SAP system acronyms:
+
+ABAP (Advanced Business Application Programming)::
+SAP proprietary programming language and part of the NetWeaver platform for building business applications.
+
+ALE (Application Link Enabling)::
+Sets up distributed applications and integrated installation of SAP systems. This involves business-driven message exchange using consistent data across loosely linked SAP apps. Apps are integrated through synchronous and asynchronous communication, rather than by use of a central database.
+
+BAPI (Business Application Programming Interface)::
+The function module that fulfills design criteria, such as:
+** Implement a method of a SAP business object
+** Maintain a static interface through different versions of the SAP system
+** Enable remotely
+** Run to completion with or without user interaction
+** Handle errors
+
+FM (function module)::
+Procedures defined in the ABAP language of SAP that allow the encapsulation and reuse of global functions in SAP.
+
+IDoc (intermediate document)::
+Standard SAP format for electronic data interchange between SAP systems. Different message types (such as delivery notes or purchase orders) generally correspond to different special formats, known as IDoc types. Multiple message types with related content can, however, be assigned to a single IDoc type.
+
+Java API::
+Handles dynamic metadata lookup and caching. The Java API implements the `JCO.Function` container used for parameters and/or tables for the SAP BAPI function module in Java. Java apps are built on top of the Java API.
+
+JNI (Java Native Interface)::
+JNI is a layer in the SAP Connector library. JCo wraps the C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
```

Comment:
Different from other definitions by repeating the term: nec?
Also, should we define JCo?
```suggestion
JNI is a layer in the SAP Connector library. JCo wraps the C libraries in Java to provide platform-native access to the SAP system. RFC middleware uses the RFC library through the JNI layer.
```